src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/VacancyStatus.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿
+@using SFA.DAS.EAS.Portal.Client.Types;
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
+@{
+    Vacancy vacancy = null;
+
+    var actionLinkText = "";
+    int? numberOfApplications = null;
+
+    if (Model.HasSinglePendingReviewVacancy)
+    {
+        vacancy = Model.AccountViewModel.Vacancies.First(v => v.Status == VacancyStatus.PendingReview);
+        actionLinkText = "Preview vacancy";
+    };
+}
+<h3 class="das-panel__heading">Your vacancy</h3>
+<p>@vacancy.TrainingTitle <span class="das-!-display--block das-text--muted">VAC @vacancy.Reference</span></p>
+<dl class="das-definition-list das-definition-list--inline">
+    <dt class="das-definition-list__title">Closing date</dt>
+    <dd class="das-definition-list__definition">@(vacancy.ClosingDate.HasValue ? vacancy.ClosingDate.Value.ToGdsFormat() : "-")</dd>
+
+    @{ if (numberOfApplications.HasValue)
+        {
+            <dt class="das-definition-list__title">Applications</dt>
+            <dd class="das-definition-list__definition">@numberOfApplications.Value</dd>
+        }
+    }
+
+    <dt class="das-definition-list__title">Status</dt>
+    <dd class="das-definition-list__definition">@vacancy.Status</dd>
+</dl>
+<ul class="govuk-list">
+    <li><a href="@Url.EmployerRecruitAction(vacancy.ManageVacancyUrl)" class="govuk-link govuk-link--no-visited-state">@actionLinkText</a> </li>
+    <li><a href="@Url.EmployerRecruitAction("create-vacancy/")" class="govuk-link govuk-link--no-visited-state">Create a vacancy</a> </li>
```

Comment:
If you view the journey linked to from the story, it states that if no vacancy has previously been submitted, the 'Create a vacancy' link should take the user to the 'create a vacancy' page, but that if a vacancy has previously been submitted, it should take them to the page that allows them to either clone an existing vacancy, **or** create a new one.

Long story short, we need to send them to `create-options`, rather than `create-vacancy`.

(Long story 😄: if the employer has created a full vacancy (status of submitted, pending review, live or closed), the `create-options` page will allow them to clone a vacancy or create one from scratch. We don't check the status of all the account's vacancies to see if they have a full vacancy, but that's OK, because the recruit system will redirect them to the `create-vacancy` page if they don't have a full vacancy. We leave the no vacancy panel pointing to the `create-vacancy` page, as that saves a redirect.)